docker_files/home/lisk/run.sh

```diff
@@ -1,4 +1,25 @@
 #!/bin/bash
 
 confd -backend env -onetime
-node app.js
+jq '
+def walk(f):
+  . as $in
+  | if type == "object" then
+      reduce keys_unsorted[] as $key
+        ( {}; . + { ($key):  ($in[$key] | walk(f)) } ) | f
+  elif type == "array" then map( walk(f) ) | f
+  else f
+  end;
+
+walk(
+    if type == "object" then
+        with_entries(select( .value != null and .value != {} and .value != [] and .value != "null" ))
+    elif type == "array" then
+        map(select( . != null and . != {} and .!= [] ))
+    else
+        .
+    end
+)' config.json >config.json_
+mv config.json_ config.json
+
+node app.js -c /home/lisk/lisk/config.json $@
```

Comment:
I am not sure about how docker run the entry point. But if `$@` means that it will have `-n` option default to `main net` then its good to go.

Reply:
Yes, it's main net by default, as  defined here: `CMD ["-n", "mainnet"]`

Reply:
@nazarhussain default is `-n mainnet` (see [our Dockerfile](https://github.com/LiskHQ/lisk/pull/2234/files#diff-3254677a7917c6c01f55212f86c57fbfR54) as well as the [Dockerfile reference](https://docs.docker.com/v17.09/engine/reference/builder/#cmd); this can be overriden at runtime.
When using `run.sh` bash will pass all paramters (`$@`) to the application and so `-n testnet` can be passed as parameter, for example. Note that there is no explicit default when using `run.sh`, something like the following `docker-compose.yml` snipped must be used:
```
[...]
entrypoint: ["/home/lisk/wait-for-it.sh", "db:5432", "--", "/home/lisk/run.sh"]
command: ["-n", "testnet"]
[...]
```

The examples in LiskHQ/lisk-docker will be updated accordingly.

Reply:
Please, bear in mind the idea is to have one single `docker-compose` file. I even foresee we can remove the `lisk-docker` repo and include in `lisk-core` that file. It will be easy to maintain and more standard. 

Perhaps we can add some docker env files to show some examples.

In the command line we should add a comment to highlight we pick the network in there.

Reply:
@diego-G I agree, let's do that in a separate issue/PR though.

Reply:
Please, open the issue to not to forget it and create a reference to this PR.